Makefile

```diff
@@ -63,6 +63,26 @@ lint-go:  # lints the Go files
 lint-md:   # lints the Markdown files
 	tools$/prettier$/node_modules$/.bin$/prettier -l .
 
+release:   # creates a new release
+	# cross-compile the binaries
+	goreleaser --rm-dist
+
+	# make Windows installer
+	$(MAKE) msi
+
+  # create GitHub release with files in alphabetical order
+	hub release create --draft --browse --message v7.4.0 \
```

Comment:
Why is it a draft release? And what is browse for?

Reply:
I'm creating draft releases that I manually review and publish until this script is battle-tested enough that I trust to run it fully automatically. Once a release is out there, it's impossible to remove it anymore.

Reply:
`--browse` opens the new release in a browser window for inspection and publishing.